arch/arm/mach-exynos/exynos.c

```diff
@@ -204,3 +204,31 @@ DT_MACHINE_START(EXYNOS_DT, "SAMSUNG EXYNOS (Flattened Device Tree)")
 	.dt_compat	= exynos_dt_compat,
 	.dt_fixup	= exynos_dt_fixup,
 MACHINE_END
+
+#define ODROID_MACHINE_START(name, compat)			\
```

Comment:
If it's just for having a nice /proc/cpuinfo maybe leave it like on mainline?
Each extra patch away from mainline makes it harder to maintain :)